### PR TITLE
Add Heels Touches Exercise

### PR DESCRIPTION
**Pull request by Gallantfarhan**
Added One Exercise + instruction + youtube video link + added github profile link and image of the exercise

### CODE REVIEW

File: src/DB/exerciseData.json [hunk 1]
```diff
@@ -12,6 +12,16 @@
     "gh-name": "druvkotwani",
     "image": "https://www.inspireusafoundation.org/wp-content/uploads/2023/01/inner-bicep-curl.gif"
   },
+  {
+    "exercise": "Heel Touches",
+    "instructions": ["Laying flat on your back, place your hands behind your head. Bend your knees and bring them up into
```
Comment (haldaranup) [inline src/DB/exerciseData.json]: Should be in a single line for each point and it requires a comma to add more. You can also take references from this file

**Review by Gallantfarhan — COMMENTED** [review]
i made the changes in my code, previously I was making a mistake not adding the coma's for the line breaks or points (I added coma In the code)

File: src/DB/exerciseData.json [hunk 2]
```diff
@@ -249,4 +249,16 @@
...
+  {
+    "exercise": "Heel Touches",
+    "instructions": [
+     "Laying flat on your back, place your hands behind your head. Bend your knees and bring them up into a 90-degree angle.",
+     "then lift into a crunch. Try to touch your heels to the ground while maintaining a 90 degree position in your legs and holding the static crunch.",
+     "Then, lift your legs back to the tabletop position as you breathe out. Consider.",
+     "adding ankle weights to make this move more challenging once you establish the proper form."
+    ],
+    "videoLink": "https://www.youtube.com/watch?v=PtX0z5_6sdM",
+    "gh-name": "https://github.com/Gallantfarhan",
```
Comment (mshamir11) [inline src/DB/exerciseData.json]: The convention for ```gh-name``` is just the ```GitHub username``` and is not the ```entire url```. Your name would be shown as the entire url. Plus the url to your GitHub repo would also break ([Code ref](https://github.com/gabrysia694/Gym-Junkies/blob/b1a7e6d6214e6d16dc9e6c312727efa1bfd12aa2/src/components/Card.jsx#L41)).  Could you please update that. Thank you.

**Review by mshamir11 — CHANGES_REQUESTED** [review]
[Nit] : Always have a clear commit message. Always start the commit message with capital letters. Just a convention. It looks pretty good.

Just adding this info if you are not aware. Please ignore if you already know of this. If you have long commit message and wish to add more description. Please do.

1. ```git commit```
Then you would get ```vi``` editor. You can have long description to your commit message as well. Then save changes. (press ```i``` to start writing. Once you are done. Press ```esc``` and type ```:wq``` to save and exit)

File: src/DB/exerciseData.json (hunk 1 above)
Comment (Gallantfarhan) [reply src/DB/exerciseData.json]: sorry i forget it was a new mistake as a newbie

File: src/DB/exerciseData.json (hunk 2 above)
Comment (Gallantfarhan) [reply src/DB/exerciseData.json]: oh I thought it was a GitHub entire URL

**Comment by Gallantfarhan** [comment]
Screenshot 2023-04-28 223237
 this is the error I'm getting when I'm about to commit

File: src/DB/exerciseData.json [hunk 3]
```diff
@@ -250,15 +250,13 @@
     "image": "https://www.inspireusafoundation.org/wp-content/uploads/2021/10/t-bar-row-muscles.gif"
   }
```
Comment (mshamir11) [inline src/DB/exerciseData.json:251]: You missed a comma here buddy. You could validate your json using a json validator. Either VS extension or https://jsonlint.com. That's why you were getting those deployment errors.

**Review by mshamir11 — CHANGES_REQUESTED** [review]

File: src/DB/exerciseData.json [hunk 4]
```diff
@@ -249,4 +249,14 @@
     "gh-name": "haldaranup",
     "image": "https://www.inspireusafoundation.org/wp-content/uploads/2021/10/t-bar-row-muscles.gif"
   }
+  {
+    "exercise": "Exercise Name",
```
Comment (gabrysia694) [inline src/DB/exerciseData.json:253]: And please add the actual name of the exercise like `"exercise": "Heel touches",`

**Review by gabrysia694 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**